frontend/packages/console-shared/src/components/drawer/__tests__/Drawer.spec.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Drawer from '../Drawer';
+import { DraggableCore } from 'react-draggable';
+import { CSSTransition } from 'react-transition-group';
+
+describe('DrawerComponent', () => {
+  it('should exist', () => {
+    const wrapper = shallow(<Drawer />);
+    expect(wrapper.find('div').exists()).toBe(true);
+  });
```

Comment:
Not sure this test is testing anything other than you are using a div (which is bad because it couples your test to a static implementation). Perhaps (even though I wouldn't consider this a great test...) you could do `expect(...isEmptyRender()).toBe(false)`. Although I suspect you may want to just drop the test.